### PR TITLE
feat(tikv/rocksdb,tikv/rust-rocksdb): enable bot to control the reviewing flow

### PR DESCRIPTION
**Pull request by wuhuizuo**
Signed-off-by: wuhuizuo <wuhuizuo@126.com>

### What problem does this PR solve?

Issue Number: ref #824 

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

### CODE REVIEW

File: prow/config/plugins.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ approve:
     commandHelpLink: "https://prow.tidb.net/command-help"
     pr_process_link: "https://book.prow.tidb.net/#/workflows/pr"
 lgtm:
-  - repos: [pingcap, ti-community-infra, PingCAP-QE, pingcap-inc]
+  - repos: [pingcap, tikv, ti-community-infra, PingCAP-QE, pingcap-inc]
```
Comment (wuhuizuo) [inline prow/config/plugins.yaml:25]: tikv/rust-rocksdb required just 1 LGTM.

File: prow/config/plugins.yaml [hunk 2]
```diff
@@ -40,6 +40,7 @@ lgtm:
       - tikv/pd
       - tikv/raft-engine
       - tikv/tikv
+      - tikv/rocksdb
```
Comment (wuhuizuo) [inline prow/config/plugins.yaml:43]: tikv/rocksdb needs 2 LGTMs

**Comment by wuhuizuo** [comment]
/cc @zhangjinpeng87

PTAL

**Review by zhangjinpeng87 — APPROVED** [review]
LGTM

**Pull request merged**